alembic.ini

```diff
@@ -0,0 +1,59 @@
+# A generic, single database configuration.
+
+[alembic]
+# path to migration scripts
+script_location = alembic
+
+# template used to generate migration files
```

Comment:
are these alembic files boiler plate?  can we get rid of these comments or are they useful?